packages/zk-transfer-web/src/util.ts

```diff
@@ -1,15 +1,44 @@
+import { keccak } from "ethereumjs-util";
 import screenNameGenerator from "docker-names";
 
-export const setScreenName = (screenName: string, chainId: string) => 
+export const setScreenName = (screenName: string, chainId: string) =>
   localStorage.setItem(`demo/${chainId}/thisScreenName`, screenName);
 
 export const getScreenName = (chainId: string) =>
-  localStorage.getItem(`demo/${chainId}/thisScreenName`)
-  
+  localStorage.getItem(`demo/${chainId}/thisScreenName`);
+
 export const capitalize = (str: string) => str[0].toUpperCase() + str.slice(1);
 
 export const generateScreenName = () => {
   const regex = /(\w+)_(\w+)/gi;
-  const replaceFn = (tokn: any, $1: any, $2: any) => capitalize($1) + " " + capitalize($2);
+  const replaceFn = (tokn: any, $1: any, $2: any) =>
+    capitalize($1) + " " + capitalize($2);
   return screenNameGenerator.getRandomName().replace(regex, replaceFn);
-}
\ No newline at end of file
+};
+
+export const getZkTokenName = (token: string | number) => {
+  const result = /ZkAssetTradeable-deploy(.+)/.exec(String(token));
+  if (result) {
```

Comment:
how about
`if (result.length > 1)`
or
`if (result.length === 1)`

Reply:
I meant to use `.test` not `.exec` I'll change it.

Reply:
ok. What about line 21?
If result is non-null, but has zero elements, or more than 1 element,
it is likely an error.

Reply:
Okay.